DOCUMENTATION.md

```diff
@@ -344,4 +344,78 @@ the same command as with pushing other assets to Studio:
 // Push analysis to Studio
 content-cli push bookmarks -p my-profile-name --id 73d39112-73ae-4bbe-8051-3c0f14e065ec --file studio_analysis_bookmarks_39c5bb7b-b486-4230-ab01-854a17ddbff2.json 
 ```
+
+### Data Pool push / pull commands
+
+#### Pull Data Pool
+
+In order to pull a Data Pool you can execute the following command:
+
+```node content-cli.js pull data-pool --id <replace-with-pool-id> --profile local -v2 true```
+
+* Note: The ```-v2``` flag enables you to export Data Pool data that can then later be used to 'Batch Push multiple Data Pools'.
+Without this flag, the export uses the old implementation that is not compatible with Batch Push.
+
+#### Batch Push multiple Data Pools
+
+In order to batch push a list of data pools use the following command:
+
+```node content-cli.js push data-pools --batchImport true -f ./request.json --profile dev1```
+
+* The ```request.json``` file contains the batch import JSON request.
+* The ```--batchImport``` flag enables import of multiple Data Pools together with data shared accross Data Pools
+  (for example, imported data connections, cross-pool scheduling triggers). If this flag 
+isn't used, then the old implementation is used which doesn't support the import of shared
+objects. Thus, for the old implementation, the imported Data Pools that rely on shared objects
+will not be functional.
+
+* The JSON request looks the following way:
+
+```
+{
```

Comment:
As already discussed in our meeting, the implementation will be adjusted to be more user-friendly. (ticket created: https://celonis.atlassian.net/browse/DP-1576)
For now, this request should be good for the Batch Import UI.